docs/source/cli.rst

```diff
@@ -0,0 +1,118 @@
+
+.. _cli:
+
+Command Line Interface
+=======================================
+
+.. highlight:: console
+
+When git-machete is installed it is added to your path, so it can be called from command line: `git-machete some_command some_command_options`. git-machete comes with a wide variety of commands and their options and in this site you can find their documentation.
+
+git-machete commands:
+
+* :ref:`add` -- Add a branch to the tree of branch dependencies.
```

Comment:
Nit: maybe align the `--`s horizontally if it doesn't destroy the resulting rendered text